nautobot/extras/datasources/git.py

```diff
@@ -509,8 +516,13 @@ def import_config_context(context_data, repository_record, job_result, logger):
     context_record = None
     # TODO: check context_data against a schema of some sort?
 
+    if "_metadata" not in context_data:
+        raise RuntimeError("data is missing the required `_metadata` key.")
```

Comment:
Perhaps something like: “The `data` dictionary is missing the required `_metadata` key.” 

Thought process being is data is a generic term and can be quickly misunderstood. Something similar for all of the messages

Reply:
Good thought. `data` isn't actually a key anywhere though. The full message seen by the user would be a la "Error in loading config context data from `foo.yml`: data is missing the required `_metadata` key", do you think that gives enough context?

Reply:
yes, that is helpful, thanks